cli/index.js

```diff
@@ -54,7 +55,13 @@ if (commands.includes(command)) {
             'ts-node/register',
             require.resolve(`./${command}.ts`),
         ],
-        { stdio: 'inherit' }
+        {
+            stdio: 'inherit',
+            env: {
+                ...process.env,
+                TS_NODE_PROJECT: path.join(__dirname, './tsconfig.json'),
```

Comment:
you don't need to provide the current directory (`./`) path
```js
TS_NODE_PROJECT: path.join(__dirname, 'tsconfig.json')
```

Reply:
Thanks a lot! Fixed.